src/lib/components/select/select.js

```diff
@@ -99,12 +103,15 @@ class Select extends PureComponent {
 
     return (
       <div
-        className={`${containerClassName}
-          ${stylesCss[`theme-${theme}`]} ${stylesCss.wrapper}`}
+        className={classNames(
+          containerClassName,
+          stylesCss[`theme-${theme}`],
+          stylesCss.wrapper,
+        )}
       >
         <Dropdown
           placeholder={placeholder}
-          className={`${className} ${isValidationOk() && styles.error}`}
+          className={`${className} ${isValidationOk() ? styles.error : ''}`}
```

Comment:
Same here, if you are using classNames, it is better to add the condition inside classNames, like:
`className={classNames(className, { [styles.error]: isValidationOk() })}`